vmm/src/cpu.rs

```diff
@@ -798,6 +816,8 @@ impl CpuManager {
             hypervisor: hypervisor.clone(),
             #[cfg(feature = "sev_snp")]
             sev_snp_enabled,
+            #[cfg(feature = "igvm")]
+            igvm_enabled,
```

Comment:
Do we need an addition boolean to know if IGVM enabled? My impression is for SevSnp IGVM is always required.